docs/variants/protectli_vp46xx/overview.md

```diff
@@ -37,3 +37,4 @@ For more information please refer to the references below.
     validation procedure.
 * [Test matrix](test-matrix.md) - describes validation scope used during
     Dasharo firmware validation procedure.
+* [Post-installation setup](post-install.md) -
```

Comment:
@pkubaj Either the `-` at the end is not required, or some description after should be added?

Reply:
```suggestion
* [Post-installation setup](post-install.md)
```